deepdream/.gitignore

```diff
@@ -0,0 +1 @@
+*.swp
```

Comment:
This `.gitignore` probably isn't necessary, it'll likely be covered by the `.gitignore` at the [root of the repo](https://github.com/spellrun/spell-examples/blob/master/.gitignore#L1).